packs/feats/godless-healing.json

```diff
@@ -0,0 +1,52 @@
+{
+    "_id": "tnzZvaJ97t0N9g6y",
+    "img": "icons/sundries/books/book-red-exclamation.webp",
+    "name": "Godless Healing",
+    "system": {
+        "actionType": {
+            "value": "passive"
+        },
+        "actions": {
+            "value": null
+        },
+        "category": "skill",
+        "description": {
+            "value": "<p>You recover an additional 5 Hit Points from a successful attempt to @UUID[Compendium.pf2e.actionspf2e.Item.Treat Wounds]{Treat your Wounds} or use @UUID[Compendium.pf2e.feats-srd.Item.Battle Medicine]{Battle Medicine on you. After you or an ally use Battle Medicine on you, you become temporarily immune to that Battle Medicine for only 1 hour, instead of 1 day.</p>"
```

Comment:
```suggestion
            "value": "<p>You recover an additional 5 Hit Points from a successful attempt to @UUID[Compendium.pf2e.actionspf2e.Item.Treat Wounds]{Treat your Wounds} or use @UUID[Compendium.pf2e.feats-srd.Item.Battle Medicine]Battle Medicine on you. After you or an ally use Battle Medicine on you, you become temporarily immune to that Battle Medicine for only 1 hour, instead of 1 day.</p>"
```